stories/demos/TreeView.tsx

```diff
@@ -130,19 +130,21 @@ export default function TreeView() {
               </div>
             </>
           );
-        },
-        unsafe_onCellInput(event, row) {
-          const hasChildren = row.children !== undefined;
-          if (event.key === ' ' && hasChildren) {
-            event.preventDefault();
-            dispatch({ id: row.id, type: 'toggleSubRow' });
-          }
-
-          if (event.key === 'Delete' && !hasChildren) {
-            event.preventDefault();
-            dispatch({ id: row.id, type: 'deleteSubRow' });
-          }
         }
+        // editor2Props: {
```

Comment:
I think we can handle it differently using https://www.w3.org/TR/wai-aria-practices/#gridNav_focus